### PR TITLE
[doc] Fixed typos and formatting issues in Configure Audit Logging in YSQL 

### PR DESCRIPTION
**Pull request by lizayugabyte**
Some typos and formatting issues (per Kannan, via slack). 
Also the whole file is kind of messed up.

Doc build preview: https://deploy-preview-8946--infallible-bardeen-164bc9.netlify.app/latest/secure/audit-logging/audit-logging-ysql/

### CODE REVIEW

File: docs/content/latest/secure/audit-logging/audit-logging-ysql.md [hunk 1]
```diff
@@ -182,8 +183,6 @@ Start the YugabyteDB Cluster with the following Audit logging Configuration.
     --ysql_pg_conf="pgaudit.log='DDL',pgaudit.log_level=notice,pgaudit.log_client=ON"
     ```
 
-
-
     Alternatively, go to a YSQL shell and execute the following commands.
```
Comment (ddhodge) [inline docs/content/latest/secure/audit-logging/audit-logging-ysql.md:188]: extra space

File: docs/content/latest/secure/audit-logging/audit-logging-ysql.md [hunk 2]
```diff
@@ -202,36 +201,30 @@ Open the YSQL shell (ysqlsh), specifying the `yugabyte` user and prompting for t
     $ ./ysqlsh -U yugabyte -W
     ```
```
Comment (ddhodge) [inline docs/content/latest/secure/audit-logging/audit-logging-ysql.md:203]: Spacing and indenting problems, code blocks need 'sh' 'output etc

File: docs/content/latest/secure/audit-logging/audit-logging-ysql.md [hunk 3]
```diff
@@ -182,8 +183,6 @@ Start the YugabyteDB Cluster with the following Audit logging Configuration.
     --ysql_pg_conf="pgaudit.log='DDL',pgaudit.log_level=notice,pgaudit.log_client=ON"
```
Comment (ddhodge) [inline docs/content/latest/secure/audit-logging/audit-logging-ysql.md:183]: The output looks wrong here, I guess due to the indenting; also the code blocks need to specify output or sql or sh, depending

**Review by ddhodge — COMMENTED** [review]
Formatting of the code blocks under 'Example' need fixing

**Comment by lizayugabyte** [comment]
@ddhodge Yeah, the file was messed up (initially I only fixed what was pointed out yesterday, but today I had time to really look at the whole file).

File: docs/content/latest/secure/audit-logging/audit-logging-ysql.md [hunk 4]
```diff
@@ -41,35 +41,38 @@ The goal of the YSQL audit logging is to provide YugabyteDB users with capabilit
...
-2. Eg. `SET pgaudit.log='DDL'`
-3. `SET` only affects the value used by the current session. A detailed description of the set command is illustrated in this [webpage](https://www.postgresql.org/docs/8.3/sql-set.html). 
+An alternative suggestion is to use the YB `SET` command, which essentially changes the run-time configuration parameters.
+
+For example, `SET pgaudit.log='DDL'`
+
+`SET` only affects the value used by the current session. For more information, see the [PostgreSQL documentation](https://www.postgresql.org/docs/11/sql-set.html). 
 
 
 ### Step 2. Load the `pgAudit` extension
```
Comment (ddhodge) [inline docs/content/latest/secure/audit-logging/audit-logging-ysql.md:64]: pgAudit or pgaudit? Seeing both used throughout. I think pgaudit?

File: docs/content/latest/secure/audit-logging/audit-logging-ysql.md [hunk 5]
```diff
@@ -41,35 +41,38 @@ The goal of the YSQL audit logging is to provide YugabyteDB users with capabilit
...
 
-```
-yugabyte=# CREATE EXTENSION IF NOT EXISTS pgaudit;
-CREATE EXTENSION
+```sql
+CREATE EXTENSION IF NOT EXISTS pgaudit;
 ```
 
 
 ## Customizing Audit Logging
```
Comment (ddhodge) [inline docs/content/latest/secure/audit-logging/audit-logging-ysql.md:73]: Sentence case in headings ya?

File: docs/content/latest/secure/audit-logging/audit-logging-ysql.md [hunk 6]
```diff
@@ -168,85 +174,77 @@ There is no default.
...
-    SET pgaudit.log_level=notice;
-    ```
+```shell
+SET pgaudit.log='DDL'; 
+SET pgaudit.log_client=ON;
+SET pgaudit.log_level=notice;
+```
 
 
 ### 2. Load `pgAudit` extension
```
Comment (ddhodge) [inline docs/content/latest/secure/audit-logging/audit-logging-ysql.md:200]: pgaudit?

File: docs/content/latest/secure/audit-logging/audit-logging-ysql.md [hunk 7]
```diff
@@ -41,35 +41,38 @@ The goal of the YSQL audit logging is to provide YugabyteDB users with capabilit
...
-CREATE EXTENSION
+```sql
+CREATE EXTENSION IF NOT EXISTS pgaudit;
 ```
 
 
 ## Customizing Audit Logging
 
-YSQL Audit logging can be further customized by configuring the pgAudit flags as described below.
+YSQL audit logging can be further customized by configuring the pgAudit flags as per the following table.
```
Comment (ddhodge) [inline docs/content/latest/secure/audit-logging/audit-logging-ysql.md:75]: pgaudit?

**Review by ddhodge — COMMENTED** [review]
Some minor additional comments

File: docs/content/latest/secure/audit-logging/audit-logging-ysql.md (hunk 4 above)
Comment (lizayugabyte) [reply docs/content/latest/secure/audit-logging/audit-logging-ysql.md:64]: It should be pgAudit in text and pgaudit in code. I fixed it.

File: docs/content/latest/secure/audit-logging/audit-logging-ysql.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/latest/secure/audit-logging/audit-logging-ysql.md]: Fixed the file

File: docs/content/latest/secure/audit-logging/audit-logging-ysql.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/latest/secure/audit-logging/audit-logging-ysql.md:207]: fixed everywhere

File: docs/content/latest/secure/audit-logging/audit-logging-ysql.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/secure/audit-logging/audit-logging-ysql.md:192]: fixed everywhere

File: docs/content/latest/secure/audit-logging/audit-logging-ysql.md (hunk 5 above)
Comment (lizayugabyte) [reply docs/content/latest/secure/audit-logging/audit-logging-ysql.md:73]: What do you mean?

File: docs/content/latest/secure/audit-logging/audit-logging-ysql.md (hunk 6 above)
Comment (lizayugabyte) [reply docs/content/latest/secure/audit-logging/audit-logging-ysql.md:200]: It's pgAudit if not in code

File: docs/content/latest/secure/audit-logging/audit-logging-ysql.md (hunk 7 above)
Comment (lizayugabyte) [reply docs/content/latest/secure/audit-logging/audit-logging-ysql.md:75]: It's fine

File: docs/content/latest/secure/audit-logging/audit-logging-ysql.md (hunk 5 above)
Comment (ddhodge) [reply docs/content/latest/secure/audit-logging/audit-logging-ysql.md:73]: Caps in headings - Customize audit logging vs Customize Audit Logging - we seem to tend to go with the first, though I see both styles everywhere

File: docs/content/latest/secure/audit-logging/audit-logging-ysql.md (hunk 6 above)
Comment (ddhodge) [reply docs/content/latest/secure/audit-logging/audit-logging-ysql.md:200]: OK wasn't sure - Later in text there is "To enable the pgaudit extension" and then just after "Finally, create the pgAudit extension as follows:"

**Review by ddhodge — APPROVED** [review]
LGTM

**Pull request merged**